### PR TITLE
(titus) Populating default IAM profile for titus

### PR DESCRIPTION
**Pull request by jeyrschabu**

### CODE REVIEW

**Comment by jeyrschabu** [comment]
@tomaslin @anotherchrisberry PTAL

File: app/scripts/modules/titus/serverGroup/configure/ServerGroupCommandBuilder.js [hunk 1]
```diff
@@ -72,6 +75,8 @@ module.exports = angular.module('spinnaker.titus.serverGroupCommandBuilder.servi
 
       var serverGroupName = namingService.parseServerGroupName(serverGroup.name);
 
+      console.log('serverGroupName', serverGroupName);
```
Comment (anotherchrisberry) [inline app/scripts/modules/titus/serverGroup/configure/ServerGroupCommandBuilder.js]: remove

Comment (jeyrschabu) [inline app/scripts/modules/titus/serverGroup/configure/ServerGroupCommandBuilder.js]: oops. yes

**Comment by anotherchrisberry** [comment]
LGTM after removing the console statement

**Pull request merged**